Sources/Rendering/OpenGL/ShaderProgram/index.js

```diff
@@ -184,6 +184,26 @@ export function vtkShaderProgram(publicAPI, model) {
     return true;
   };
 
+  publicAPI.setUniform4f = (name, v) => {
+    const location = publicAPI.findUniform(name);
+    if (location === -1) {
+      model.error = `Could not set uniform ${name} . No such uniform.`;
+      return false;
+    }
+    model.context.uniform4f(location, v[0], v[1], v[2], v[3]);
+    return true;
+  };
+
+  publicAPI.setUniform4fv = (name, count, v) => {
+    const location = publicAPI.findUniform(name);
+    if (location === -1) {
+      model.error = `Could not set uniform ${name} . No such uniform.`;
+      return false;
+    }
+    model.context.uniform4fv(location, count, v);
```

Comment:
@martinken Sorry I am confused. Do you agree that `count` parameter must be removed here?